docs/src/main/paradox/docs/delta/api/current/views/elasticsearch-view-api.md

```diff
@@ -28,30 +30,39 @@ The last stage takes the JSON document, generated through the pipeline steps, an
   "sourceAsText": {sourceAsText},
   "includeMetadata": {includeMetadata},
   "includeDeprecated": {includeDeprecated},
-  "mapping": _elasticsearch mapping_
+  "mapping": _elasticsearch mapping_,
+  "settings": _elasticsearch settings_,
+  "permission": "views/query"
 }
 ```
 
 where...
 
-- `{resourceSchema}`: Iri - Selects only resources that are validated against the provided schema Iri. This field is optional.
+- `{resourceSchema}`: Iri - Selects only resources that are validated against the provided schema Iri. This field is
+  optional.
 - `{resourceType}`: Iri - Select only resources of the provided type Iri. This field is optional.
 - `{tag}`: String - Selects only resources with the provided tag. This field is optional.
-- `_elasticsearch mapping_`: Json object - Defines the value types for the Json keys, as stated at the 
-  @link:[ElasticSearch mapping documentation](https://www.elastic.co/guide/en/elasticsearch/reference/current/indices-put-mapping.html#indices-put-mapping){ open=new }.
-- `{sourceAsText}`: Boolean - If true, the resource's payload will be stored in the ElasticSearch document as a single 
-  escaped string value of the key `_original_source`. If false, the resource's payload will be stored normally in the ElasticSearch document. The default value is `false`.
-- `{includeMetadata}`: Boolean - If true, the resource's nexus metadata (`_constrainedBy`, `_deprecated`, ...) will be 
+- `_elasticsearch mapping_`: Json object - Defines the value types for the Json keys, as stated at the
+  @link:[ElasticSearch mapping documentation](https://www.elastic.co/guide/en/elasticsearch/reference/current/indices-put-mapping.html#indices-put-mapping){
+  open=new }.
+- `_elasticssearch settings_`: Json object(Optional) - defines Elasticsearch
+  @link:[index settings](https://www.elastic.co/guide/en/elasticsearch/reference/current/indices-create-index.html#create-index-settings){
+  open=new } for the underlying Elasticsearch index. Default settings are applied, if not specified.
+- `{sourceAsText}`: Boolean - If true, the resource's payload will be stored in the ElasticSearch document as a single
+  escaped string value of the key `_original_source`. If false, the resource's payload will be stored normally in the
+  ElasticSearch document. The default value is `false`.
+- `{includeMetadata}`: Boolean - If true, the resource's nexus metadata (`_constrainedBy`, `_deprecated`, ...) will be
   stored in the ElasticSearch document. Otherwise it won't. The default value is `false`.
 - `{includeDeprecated}`: Boolean - If true, deprecated resources are also indexed. The default value is `false`.
 - `{someid}`: Iri - The @id value for this view.
+- `permission`: String(Optional) - permission required to query this view. Defaults to `views/query`.
```

Comment:
The fact that you write `String(Optional)` here seems correct to me, but it is not consistent with the rest of the documentation. Either we change it everywhere else, or you align this one